### PR TITLE
Add Prometheus metrics for service creation

### PR DESCRIPTION
**Pull request by terrytangyuan**
Part of #22.  /cc @ywskycn who proposed to add this originally.

### CODE REVIEW

File: pkg/controller.v1/common/service.go [hunk 1]
```diff
@@ -28,10 +28,23 @@ import (
...
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/kubernetes/pkg/controller"
 )
 
+var (
+	succeededServiceCreationCount = promauto.NewCounter(prometheus.CounterOpts{
```
Comment (Jeffwan) [inline pkg/controller.v1/common/service.go:38]: Do we want to have a rough plan to discuss what kinds of metrics we want to bring to kubeflow/common?

Comment (terrytangyuan) [reply pkg/controller.v1/common/service.go:38]: Yes I’ll put together a list.

**Pull request merged**